yastd/dbus_api.md

```diff
@@ -0,0 +1,29 @@
+
+## get the list of available languages
```

Comment:
just to clarify it is available languages for installation or for clients?

Reply:
Huh, good point. This is for the installed system.
The UI at installation time should be in the same language, but the D-Bus API is a lower layer and does not care what its caller is showing to the user.

Reply:
@imobachgs can you please clarify this? I think our initial goal is to have API for installer. So what exactly should do that list of available languages?

Reply:
The idea is to set the language for the installed system. We most probably should sync the UI language too, but that's out of D-Bus control by now.